packages/devtools_app/lib/src/logging/checkout_box.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:html_shim/html.dart' as html;
+
+import '../../src/ui/html_elements.dart';
+import 'config.dart';
+
+class ToolBarCheckbox {
+  ToolBarCheckbox(this.extensionDescription) : element = CoreElement('label') {
+    final checkbox = CoreElement('input')..setAttribute('type', 'checkbox');
+    _checkboxElement = checkbox.element;
+
+    element.add(<CoreElement>[
+      checkbox,
+      span(text: ' ${extensionDescription.name}'),
+    ]);
+
+    _checkboxElement.checked = extensionDescription.enabled;
+    _checkboxElement.onChange.listen((_) {
+      final bool selected = _checkboxElement.checked;
+      config[extensionDescription.tag] = selected;
+    });
+  }
+
+  final ToolBarCheckboxDescription extensionDescription;
```

Comment:
this variable name is misleading.  This isn't anything to do with service extensions.